src/components/artifacts/ArtifactsView.tsx

```diff
@@ -192,9 +208,17 @@ class ArtifactsView extends React.Component<Props> {
     return (
       <Paper elevation={1}>
         <Toolbar className={classes.title}>
-          <Typography variant="h6" color="inherit">
+          <Typography variant="h6" color="inherit" className={classes.title}>
             {this._currentPath() || 'Artifacts'}
           </Typography>
+          {this._getSelectedArtifact() === null ? null : (
+            <Button
```

Comment:
I meant these toggle buttons https://material-ui.com/components/toggle-button/#toggle-buttons

Reply:
Ohhh, that might help.  I can do this later, I've got a few other projects needing bug fixes in the meantime 😄

Reply:
Done!